core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -329,15 +329,15 @@ protected function doReplace($strBuffer, $blnCache)
 
 						if ($rgxp == 'date')
 						{
-							$arrCache[$strTag] = Date::parse(Config::get('dateFormat'), $value);
+							$arrCache[$strTag] = Date::parse($objPage->dateFormat, $value);
```

Comment:
Should we use the config as a fall back if `$objPage` does not exist?
```suggestion
							$arrCache[$strTag] = Date::parse($objPage->dateFormat ?? Config::get('dateFormat'), $value);
```

Reply:
Or maybe fallback to `null` because the date class handles the fallback itself?
```suggestion
							$arrCache[$strTag] = Date::parse($objPage->dateFormat ?? null, $value);
```

Reply:
I was thinking about the same, but the rest of the code currently already relies on `$objPage` not being null. Though generally I am in favour of doing it this way.

Reply:
I see, the `date` tag already seems to rely on the `$objPage` in the cache handling section. So I guess it’s OK then.